lib/ansible/plugins/callback/default.py

```diff
@@ -335,11 +336,12 @@ def v2_runner_item_on_skipped(self, result):
             self._display.display(msg, color=C.COLOR_SKIP)
 
     def v2_playbook_on_include(self, included_file):
-        msg = 'included: %s for %s' % (included_file._filename, ", ".join([h.name for h in included_file._hosts]))
-        label = self._get_item_label(included_file._vars)
-        if label:
-            msg += " => (item=%s)" % label
-        self._display.display(msg, color=C.COLOR_SKIP)
+        if self.display_includes:
```

Comment:
This should use the option from config rather than relying on the backwards compatible properties.

```suggestion
        if self.get_option('display_includes'):
```